tests/admin_views/tests.py

```diff
@@ -1912,6 +1912,102 @@ def test_change_view_save_as_new(self):
         new_article = Article.objects.latest('id')
         self.assertRedirects(post, reverse('admin:admin_views_article_change', args=(new_article.pk,)))
 
+    def test_change_view_with_view_only_inlines(self):
+        """
+        User with change permission to a section but view-only for inlines.
+        """
+        self.viewuser.user_permissions.add(get_perm(Section, get_permission_codename('change', Section._meta)))
+        self.client.force_login(self.viewuser)
+
```

Comment:
Generally I consider the comment enough of a separator that blank lines aren't needed.